IDE/XCODE-FIPSv2/macOS-C++/Intel/user_settings.h

```diff
@@ -650,12 +650,15 @@ extern "C" {
 //#define NO_SIG_WRAPPER
 
 /* wolfSSL engineering ACVP algo and operational testing only (Default: Off) */
-#if 0
+#if 0 /* Automation Grep Target */
```

Comment:
What does this mean?

Reply:
I needed a unique-ness to the line so that when this is tested in Jenkins I have a regular expression to match on to flip this `#if 0` section to `#if 1` to run the harness. Thanks!

Reply:
Example of above explanation:
```
# Turn on the features necessary to build/run op-test and harness
NEW_LINE_M1="#if 1 /* Automation Update Successful */"
USR_STTNGS_M1=IDE/XCODE-FIPSv2/macOS-C++/M1/user_settings.h
if ! sed -i.bak "s@^#if 0.*Automation Grep Target.*@${NEW_LINE_M1}@" "$USR_STTNGS_M1"; then
    echo "Failed to update user_settings.h (M1)"
    exit 1
fi
```